pkgs/stdenv/generic/make-derivation.nix

```diff
@@ -256,21 +256,19 @@ in rec {
 
           mesonFlags = if mesonFlags == null then null else let
             # See https://mesonbuild.com/Reference-tables.html#cpu-families
-            cpuFamilies = {
-              aarch64  = "aarch64";
-              armv5tel = "arm";
-              armv6l   = "arm";
-              armv7l   = "arm";
-              i686     = "x86";
-              x86_64   = "x86_64";
-            };
+            cpuFamily = platform: with platform;
+              /**/ if isAarch64 then "arm"
+              else if isAarch32 then "aarch64"
```

Comment:
this looks wrong

Reply:
Ah, shit!

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/86283